plio/settings.py

```diff
@@ -322,3 +322,22 @@
     "location": os.environ.get("BIGQUERY_LOCATION", ""),
     "credentials": os.environ.get("BIGQUERY_CREDENTIALS", ""),
 }
+
+SENTRY_DSN = os.environ.get("SENTRY_DSN", None)
+
+if APP_ENV in ["staging", "production"] and SENTRY_DSN is not None:
+    import sentry_sdk
+    from sentry_sdk.integrations.django import DjangoIntegration
+
+    # Refer Sentry documentation for more configs: https://docs.sentry.io/platforms/python/configuration/
+    sentry_sdk.init(
+        dsn=SENTRY_DSN,
+        integrations=[DjangoIntegration()],
+        # Set traces_sample_rate to 1.0 to capture 100%
+        # of transactions for performance monitoring.
+        traces_sample_rate=1.0,
+        # If you wish to associate users to errors (assuming you are using
+        # django.contrib.auth) you may enable sending PII data.
+        send_default_pii=True,
```

Comment:
exactly which params are passed?

Reply:
@dalmia it seems like we may want to remove this option. While it gives more info on the error, we may be violating user privacy.

This link in the documentation gives more idea about what PII includes:
https://docs.sentry.io/platforms/python/guides/django/data-management/sensitive-data/

What do you think?

Reply:
Initially, as it was in the example snippet I was using, it felt like a good thing. But upon reading more, it seems either we need to find a middle ground or not use it.

Maybe we can go without this option and see if we're having difficulty replicating issues. If not, then we're good without it. If yes, we can check up to what level of PII we need and how to configure that.

Reply:
Maybe let's just send the user's DB id and scrub everything else?

Reply:
By default it sends user id, email and username. Refer function `_set_user_info` in this file below:
https://github.com/getsentry/sentry-python/blob/b0658904925ec2b625b367ae86f9762b5a382d5f/sentry_sdk/integrations/django/__init__.py#L451


I've added a `strip_sensitive_data` function that removes email and username.